lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -10,7 +10,6 @@ class Base
           ::Errno::ENOENT,
           ::Racc::ParseError,
           ::RubyParser::SyntaxError,
-          ::RuntimeError,
```

Comment:
@wfleming ah okay, I may have misclassified this error. What else might have prevented this error from halting analysis?:
```
Exception in thread "Ruby-0-Thread-3: /usr/src/app/lib/cc/engine/analyzers/file_thread_pool.rb:20" java.lang.OutOfMemoryError: Java heap space
	at java.nio.HeapCharBuffer.<init>(HeapCharBuffer.java:57)
	at java.nio.CharBuffer.allocate(CharBuffer.java:335)
	at org.jruby.RubyEncoding$UTF8Coder.<init>(RubyEncoding.java:304)
	at org.jruby.RubyEncoding.getUTF8Coder(RubyEncoding.java:365)
	at org.jruby.RubyEncoding.decodeUTF8(RubyEncoding.java:280)
	at org.jruby.runtime.Helpers.decodeByteList(Helpers.java:2642)
	at org.jruby.RubyString.decodeString(RubyString.java:759)
	at org.jruby.RubyString.toString(RubyString.java:750)
	at org.jruby.RubyString.asJavaString(RubyString.java:1276)
	at usr.src.app.lib.cc.engine.analyzers.php.ast.RUBY$block$json_to_ast$7(/usr/src/app/lib/cc/engine/analyzers/php/ast.rb:40)
```